tensorflow/lite/micro/micro_interpreter.cc

```diff
@@ -175,6 +176,28 @@ TfLiteStatus MicroInterpreter::PrepareNodeAndRegistrationDataFromFlatbuffer() {
       node->builtin_data = reinterpret_cast<void*>(builtin_data);
       node->custom_initial_data = custom_data;
       node->custom_initial_data_size = custom_data_size;
+
```

Comment:
Do you still need this code? It looks like all of the 'eval' code uses TFLiteEvalTensors now, so I'm hoping we can delete this since I think it's the main outstanding change that we'd like to avoid in the core framework code.

Reply:
Prepare() function of unidirectional_sequence_lstm.cc  is still using TFLiteTensor and using intermediates to get quantization scale and zeropoint

Reply:
I'm hoping you should be able to use context->GetTensor() in prepare, which doesn't use context->tensors, but instead allocates a temporary from the arena:
https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/micro/micro_interpreter.cc#L349

Reply:
@petewarden  
Please review the below micro interpreter code and I assume we need to add op->intermediates() to node->intermediates after converting to TfLiteIntArray

Reply:
Thanks! That seems a lot more minimal, I think the change should be fine now. Do you still need the include line at the top?

```
#include "tensorflow/lite/micro/kernels/xtensa/lstm/helper_utils/util.h"
```

Reply:
As this OpUsesIntermediates() function is implemented at tensorflow/lite/micro/kernels/xtensa/lstm/helper_utils/util.h
do you want me to move this implementation to any specific existing header?

Reply:
Can you check whether the length of the op->intermediates array is more than zero, instead of checking for specific ops?

Reply:
@petewarden 
Added op->intermediates array is more than zero , and this and removed header
#include "tensorflow/lite/micro/kernels/xtensa/lstm/helper_utils/util.h"

Reply:
Thanks, this part of the change looks good now.